tests/integration/test_backup_restore_on_cluster/test_different_versions.py

```diff
@@ -0,0 +1,125 @@
+import random
```

Comment:
I have problems on this test right now: I extened the backup meta file for lightweight backup, and update the file version to V2. The `old_node` will not recognize a v2 meta file, that means as long as we upgrade the file version, this test has to fail ...

Any suggestions?